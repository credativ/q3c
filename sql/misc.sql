select q3c_dist(1,2,3,4);
select q3c_dist(null,null,3,4);
select q3c_sindist(1,2,3,4);
select q3c_sindist(null,null,3,4);
select q3c_dist_pm(1,2,3,4,1,2000,1,0,2005);
select q3c_sindist_pm(1,2,3,4,1,2000,1,0,2005);
select q3c_dist_pm(1,2,3,4,1,2000,null,0,2005);
select q3c_sindist_pm(1,2,3,4,1,2000,null,0,2005);
select q3c_dist_pm(1,2,3,null,1,2000,0,0,2005);
select q3c_sindist_pm(1,2,3,null,1,2000,0,0,2005);
