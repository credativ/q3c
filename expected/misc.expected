    round     
--------------
 2.8264172167
(1 row)

 round 
-------
      
(1 row)

    round     
--------------
 0.0006082459
(1 row)

 round 
-------
      
(1 row)

    round     
--------------
 2.0000055556
(1 row)

    round     
--------------
 0.0003045882
(1 row)

 round 
-------
      
(1 row)

 round 
-------
      
(1 row)

    round     
--------------
 2.2359771471
(1 row)

    round     
--------------
 0.0003806925
(1 row)

